lib/bademagic_module/utils/image_utils.dart

```diff
@@ -1,32 +1,40 @@
 import 'dart:ui' as ui;
 import 'dart:ui';
+import 'package:badgemagic/bademagic_module/utils/converters.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_svg/flutter_svg.dart';
 
 class ImageUtils {
-  // Define target width and height for the bitmap
-  static double targetHeight = 11.0;
-  static double targetWidth = 44.0;
+  late double originalHeight;
+  late double originalWidth;
 
-  //function to load and scale the svg according to the badge size
-  Future<ui.Image> scaleSVG(String svg) async {
+  late ui.Picture picture;
```

Comment:
Sharing the picture when not designing the class to handle per image operations would be confusing. If this class is going to process one image anyway. Remove gateways to loadSVG, scaleSVG, etc by making them private